src/components/StoryJumbo.vue

```diff
@@ -25,7 +25,8 @@
             :padding="['none', '2x']"
             p="1"
             alignment="centered"
-          >Storyscript is a top-level programming language and platform that is specially designed for data flow between highly-reusable components (e.g. microservices), written in any other programming language. Bash for the cloud. Best of all, it's open source!</s-text>
+          >The cloud monolith backed by a “serverless” microservice architecture for rapid backend application development.
```

Comment:
@stevepeak I really prefer not having the word monolith in there. Here's a copy without that word:
```suggestion
          >The distributed cloud architecture for rapidly building backend applications backed by “serverless” microservices and functions.
```